ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -820,6 +818,30 @@ tests:
         OPENSHIFT_VERSION=4.13
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-static-ip-suite-4-14
+  optional: true
+  run_if_changed: ^(data/.*)$
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        STATIC_IPS=true
+        TEST_FUNC=test_networking
+        OPENSHIFT_VERSION=4.14
+      PACKET_PLAN: m3.large.x86
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-bond-4-14
+  optional: true
+  run_if_changed: ^(data/.*)$
```

Comment:
should we really run it each time data is updated? Or should we just keep it as a manually triggered test (`always_run: false`)?

Reply:
Not sure, in my mind it would make sense to have it periodic (but indeed maybe not every time `data` is changed). This way we can catch possible regressions. Is it worth the cost though? Are manual test run for regression each time?

Reply:
As discussed we will keep presubmit on-demand tests plus periodic for testing 4.14 + both network types